nixos/modules/services/web-apps/hedgedoc.nix

```
@@ -1,1001 +1,254 @@
 { config, lib, pkgs, ... }:
-
-with lib;
-
 let
+  inherit (lib) versionAtLeast mkEnableOption mkOption types mdDoc mkIf
+    optionals mkPackageOption mkRemovedOptionModule mkDefault
+    mkRenamedOptionModule literalExpression maintainers;
+
   cfg = config.services.hedgedoc;
 
   # 21.03 will not be an official release - it was instead 21.05.  This
   # versionAtLeast statement remains set to 21.03 for backwards compatibility.
   # See https://github.com/NixOS/nixpkgs/pull/108899 and
   # https://github.com/NixOS/rfcs/blob/master/rfcs/0080-nixos-release-schedule.md.
-  name = if versionAtLeast config.system.stateVersion "21.03"
+  name =
+    if versionAtLeast config.system.stateVersion "21.03"
     then "hedgedoc"
     else "codimd";
 
-  settingsFormat = pkgs.formats.json {};
-
-  prettyJSON = conf:
-    pkgs.runCommandLocal "hedgedoc-config.json" {
-      nativeBuildInputs = [ pkgs.jq ];
-    } ''
-      jq '{production:del(.[]|nulls)|del(.[][]?|nulls)}' \
-        < ${settingsFormat.generate "hedgedoc-ugly.json" cfg.settings} \
-        > $out
-    '';
+  settingsFormat = pkgs.formats.json { };
 in
 {
+  meta.maintainers = with maintainers; [ h7x4 ];
```

```suggestion
  meta.maintainers = with maintainers; [ SuperSandro2000 h7x4 ];
```
see https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/servers/web-apps/hedgedoc/default.nix#L81 , everything doesn't make sense

---

Not entirely sure what you meant by "everything doesn't make sense", but sure, I'll add you as maintainer.